resume.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <link href='http://fonts.googleapis.com/css?family=Lato:300,700,300italic,700italic' rel='stylesheet' type='text/css'/>
+        <link rel="stylesheet" type="text/css" href="css/styles.css">
+        <title>Resume</title>
+    </head>
+
+    <body>
+        <h2>Notable Credits</h2>
+
+        <ul>
+            <li>Agents of S.H.I.E.L.D., Creator (2013-present)</li>
+            <li>Avengers: Age of Ultron, Writer (2015)</li>
+            <li>In Your Eyes, Writer (2014)</li>
+            <li>Much Ado About Nothing, Screenwriter (2012)</li>
+            <li>The Avengers, Screenwriter/Story Developer (2012)</li>
+            <li>The Cabin in the Woods, Writer (2012)</li>
+            <li>Dollhouse, Creator (2009-2010)</li>
+            <li>Dr. Horrible’s Sing-Along Blog, Writer (2008)</li>
+            <li>Angel, Creator (1999-2004)</li>
+            <li>Firefly, Creator (2004)</li>
+            <li>Buffy the Vampire Slayer, Creator (1997-2003)</li>
+            <li>Alien: Resurrection, Writer (1997)</li>
+            <li>Toy Story, Screenwriter (1995)</li>
+            <li>Roseanne, Writer (1989-1990)</li>
+        </ul>
+
+        <img src="images/joss_whedon_long.jpg" alt="Intense Look" class="long" title="Hire Me">
```

Comment:
@eloisebarrow The image can not be viewed because the the file extension of the actual file name is in caps. joss_whedon_long `.JPG`.  How ever you are calling  joss_whedon_long `.jpg`. In the future always try to keep all of your file names in lowercase. 🔨